lib/ex_auction/oauth2/endpoints/bid/handler.ex

```diff
@@ -0,0 +1,13 @@
+defmodule ExAuction.Bid.Handler do
+  @moduledoc """
+  Handles the /login endpoint logic
+  """
+  alias ExAuction.GoogleClient
+
+  def init, do: :ok
+
+  def bid(payload) do
+    # Here should be executed the logic
```

Comment:
I would delegate it even further e.g. have a dedicated app so that we have a nice separation via an api